src/api/src/lib.rs

```diff
@@ -31,6 +31,12 @@ impl Cli {
                     .takes_value(true)
                     .help("vCPU configuration.\n\tFormat: \"num=<u8>\""),
             )
+            .arg(
```

Comment:
I think for now it's better to not export it at the API level as well because we don't have a customer need for it yet. Going forward we want to have a VMM library, and we can export it in the VMM interface, but for the API we should just keep it simple.